### PR TITLE
update bpmn process id

### PR DESCRIPTION
**Pull request by christinaausley**
## Description

bpmnProcessId has been renamed as processDefinitionId

See related [Slack thread](https://camunda.slack.com/archives/C06UKS51QV9/p1727944038036749) and [discussion](https://camunda.slack.com/archives/C0484UFR2VA/p1728046031821759).

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [x] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [ ] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [ ] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [x] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

File: api/camunda/camunda-openapi.yaml [hunk 1]
```diff
@@ -648,7 +648,7 @@ paths:
       description: |
         Creates and starts an instance of the specified process.
         The process definition to use to create the instance can be specified either using its unique key
-        (as returned by Deploy resources), or using the BPMN process ID and a version.
```
Comment (jwulf) [inline api/camunda/camunda-openapi.yaml:651]: Good catch!

File: api/camunda/camunda-openapi.yaml [hunk 2]
```diff
@@ -1889,9 +1889,9 @@ components:
         endDate:
           type: string
           format: date-time
-        bpmnProcessId:
```
Comment (jwulf) [inline api/camunda/camunda-openapi.yaml:1892]: @nicpuppa - should this be here?

File: api/camunda/camunda-openapi.yaml [hunk 3]
```diff
@@ -1889,9 +1889,9 @@ components:
...
+        processDefinitionId:
           type: string
-          description: The bpmn process ID.
+          description: The process desfinition ID.
```
Comment (jwulf) [inline api/camunda/camunda-openapi.yaml:1894]: Typo

File: api/camunda/camunda-openapi.yaml [hunk 4]
```diff
@@ -1889,9 +1889,9 @@ components:
         endDate:
           type: string
           format: date-time
-        bpmnProcessId:
+        processDefinitionId:
```
Comment (jwulf) [inline api/camunda/camunda-openapi.yaml:1892]: @christinaausley I'm not sure we can make this change here - this is the underlying specification and describes the actual operation of the API.

File: api/camunda/camunda-openapi.yaml (hunk 2 above)
Comment (jwulf) [reply api/camunda/camunda-openapi.yaml:1892]: Looks like has been updated here: https://github.com/camunda/camunda/blob/main/zeebe/gateway-protocol/src/main/proto/rest-api.yaml#L2414

File: api/camunda/camunda-openapi.yaml [hunk 5]
```diff
@@ -1935,7 +1935,7 @@ components:
         processDefinitionVersion:
           type: integer
           format: int32
-        bpmnProcessId:
+        processDefinitionId:
```
Comment (pepopowitz) [inline api/camunda/camunda-openapi.yaml:1938]: :x: This is another actual field name change. I don't think this should be here, unless we're certain it's in the actual 8.6 api.

**Comment by pepopowitz** [comment]
I added a `hold` label to this PR, because from Slack it sounds like these functional changes are not in place at the source yet, at least not for the 8.6 release. 

If/when we hear that the 8.6 release has been updated with spec changes, I will come back here and update the spec accordingly.

**Review by pepopowitz — COMMENTED** [review]
I noted one more field name change, which would break the docs for users if

**Comment by christinaausley** [comment]
Closing based on conversation [here](https://camunda.slack.com/archives/C0484UFR2VA/p1728046031821759).

**Pull request closed without merging**